api/src/pcapi/core/finance/api.py

```diff
@@ -1495,11 +1502,64 @@ def _mark_as_processed(pricings: sqla_orm.Query) -> None:
                     continue
 
                 total = pricings.with_entities(sqla.func.sum(models.Pricing.amount)).scalar()
-                if not total:
+
+                all_current_incidents = (
+                    models.FinanceIncident.query.join(models.FinanceIncident.booking_finance_incidents)
+                    .join(models.BookingFinanceIncident.finance_events)
+                    .join(models.FinanceEvent.pricings)
+                    .outerjoin(models.Pricing.cashflows)
+                    .filter(
+                        models.FinanceIncident.venueId.in_(venue_ids),
+                        models.FinanceIncident.status == models.IncidentStatus.VALIDATED,
+                        models.Cashflow.id.is_(None),  # Add this line to filter incidents with cashflows
+                    )
+                    .all()
+                )
+
+                if not total or (total > 0):
+                    last_cashflow_age = 0
+
+                    if total > 0:
```

Comment:
[détail / doc] J'ajouterais un commentaire pour aider à la compréhension de cette condition :

```suggestion
                    # The total is positive if the pro owes us more than we do.
                    if total > 0:
```